resources/views/users/items/add.blade.php

```diff
@@ -120,4 +122,28 @@
     </form>
     @include('users.categories.modals.create_category')
 </div>
+
```

Comment:
Separate scripts from blade file to js file

Reply:
I have created new js files inside public holder. Please check it.